modules/ROOT/pages/endpoint-as2-send.adoc

```diff
@@ -94,6 +94,18 @@ a|Whether you want the partner to return an MDN receipt, which confirms the rece
 |MDN response
 |The AS2 messages sent from Partner Manager always request the receiver to send a Synchronous MDN receipt back. The default is `Sync`.
 | Yes
+
+|Authentication type
+a| Authentication type used to access the endpoint. Allowed values:
+
+* None
++
+Does not require backend applications to authenticate when connecting to this endpoint.
```

Comment:
AS2 send endpoints are interfaces with partners and not backend applications. We should change this as below.
"No additional authentication is requires to send messages to the partner's AS2 endpoint"